numpy/random/mtrand/mtrand.pyx

```diff
@@ -693,14 +711,18 @@ cdef class RandomState:
 
         """
         cdef ndarray state "arrayObject_state"
+        version_info = (self.version,) if self.version else ()
+        rng_info = "MT19937"
         state = <ndarray>np.empty(624, np.uint)
         with self.lock:
-            memcpy(<void*>PyArray_DATA(state), <void*>(self.internal_state.key), 624*sizeof(long))
+            memcpy(<void*>PyArray_DATA(state),
+                   <void*>(self.internal_state.key),
+                   624 * sizeof(long))
             has_gauss = self.internal_state.has_gauss
             gauss = self.internal_state.gauss
             pos = self.internal_state.pos
         state = <ndarray>np.asarray(state, np.uint32)
-        return ('MT19937', state, pos, has_gauss, gauss)
+        return version_info + (rng_info, state, pos, has_gauss, gauss)
```

Comment:
This doesn't match the docstring, which claims it returns nested tuples. I reckon the docstring format is the better one, so go with that.